src/compiler/scala/tools/nsc/ast/DocComments.scala

```diff
@@ -133,8 +134,12 @@ trait DocComments { self: Global =>
     mapFind(sym :: allInheritedOverriddenSymbols(sym))(docComments get _)
 
   /** The cooked doc comment of an overridden symbol */
-  protected def superComment(sym: Symbol): Option[String] =
-    allInheritedOverriddenSymbols(sym).iterator map (x => cookedDocComment(x)) find (_ != "")
+  protected def superComment(sym: Symbol): Option[String] = {
+    val getter = sym.getterIn(sym.owner)
```

Comment:
There's `sym.getter` as a shorthand

Reply:
Sadly that doesn't work (see the stricken text in the PR description): evidently Scala can't decide whether to eta-expand the deprecated one-arg `getter` method or select from the one you mention.

Reply:
Oh, overloading resolution..!
```
scala> :pa -raw
// Entering paste mode (ctrl-D to finish)

class A { def f = 0 }
object C extends A { def f(x: Int) = 1 }

// Exiting paste mode, now interpreting.


scala> C.f
<console>:12: error: ambiguous reference to overloaded definition,
both method f in object C of type (x: Int)Int
and  method f in class A of type => Int
match expected type ?
       C.f
         ^
```

It works if you provide the `: Symbol` ascription.

Reply:
Yeah. I think I'm going to pop over to sbt and see if I can get rid of them using the deprecated one.